classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -59,6 +60,12 @@ public function boot(Application $app)
             return SpeakerAccess::userHasAccess($app);
         };
 
+        $csrfChecker = function (Request $req) use ($app) {
```

Comment:
Can we rename `$req` to `$request` here? There's no real advantage of shortening it.